core/src/test/java/org/mule/tck/junit4/AbstractMuleTestCase.java

```diff
@@ -55,6 +55,11 @@
     public static final String TEST_TIMEOUT_SYSTEM_PROPERTY = "mule.test.timeoutSecs";
 
     /**
+     * Default timeout for waiting for responses
+     */
+    public static final int RECEIVE_TIMEOUT = 5000;
```

Comment:
Agree, but 5000 is also far to long for AbstractMuleContextTestCase IMO. Anyway i'll revert this and use a local latch timeout for this instead.